libs/shared/providers/src/txTracker/utils.ts

```diff
@@ -0,0 +1,38 @@
+/* eslint-disable react-hooks/rules-of-hooks */
+import { prepareWriteContract as prepareWriteContractOrig } from '@wagmi/core';
+import { keccak256, toBytes } from 'viem';
+
+const defaultSuffix = keccak256(toBytes('oeth.com')).slice(2, 10); // 9fed593b
```

Comment:
Maybe pass the default suffix as a param to the hook so this can be reused on other apps

Reply:
👍